feat(changelog): object-storage-added-improved-user-experience-for 2023-02-03 (#1263)
scaleway/docs-content: changelog/february2023/2023-02-03-object-storage-added-improved-user-experience-for.mdx

---

> title: Improved User experience for bucket deletion from the Console

Reviewer: ```suggestion
title: Improved user experience for bucket deletion from the console
```

status: added
author:
    fullname: 'Join the #object-storage channel on Slack.'

>     url: 'https://scaleway-community.slack.com/archives/CEB64PNGY'

Reviewer 2: ```suggestion
    url: 'https://slack.scaleway.com'
```

date: 2023-02-03
category: storage
product: object-storage
---

> Buckets deleted from the Console are deleted after 24h. The UX now reflects if the name is already taken by a bucket whose deletion is pending.

Reviewer: ```suggestion
Buckets deleted from the console are only deleted after 24h. The UX now reflects if a bucket name is unavailable because it is taken by a bucket whose deletion is pending.
```
